utils/utils.c

```diff
@@ -232,6 +235,29 @@ static bool can_remove_directory(const char *path)
 	return is_uftrace_directory(path) || is_empty_directory(path);
 }
 
+static bool create_uftrace_opts(const char *dirname)
+{
+	const char *opts_str = strv_join(&uftrace_opts, "");
```

Comment:
I think the delimiter should be a space.  Also it'd be better to bail out if `uftrace_opts` is already empty.

Reply:
Okay. I will change it.

Reply:
done.